pkg/ibmvpc/examples/out/lint_out/acl3_overlap_subnets_Lint

```diff
@@ -0,0 +1,27 @@
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 1 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 1, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.30.0/24, conn: protocol: icmp, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 2 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 2, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 5 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 5, direction: inbound , src: 10.240.30.0/24 , dst: 10.240.20.0/24, conn: protocol: tcp, srcPorts: 1-65535, dstPorts: 22-22, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 6 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 6, direction: inbound , src: 10.240.10.0/24 , dst: 10.240.20.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 0 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 0, direction: outbound , src: 0.0.0.0/0 , dst: 10.240.10.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 1 splits subnets subnet1-ky (10.240.10.0/16), subnet3-ky (10.240.30.0/24). Splitting rule details: index: 1, direction: outbound , src: 10.240.30.0/31 , dst: 10.240.20.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 2 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 2, direction: inbound , src: 10.240.10.0/24 , dst: 0.0.0.0/0, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 3 splits subnets subnet1-ky (10.240.10.0/16), subnet3-ky (10.240.30.0/24). Splitting rule details: index: 3, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.30.0/31, conn: all, action: allow
+________________________________________________________________________________________________________________________________________________________________________________________________________
+
+"Overlapping CIDR ranges between different subnets" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+subnet subnet1-ky of cidr 10.240.10.0/16 and subnet subnet2-ky of cidr 10.240.20.0/24, both from VPC test-vpc1-ky, overlap in 10.240.20.0/24
+subnet subnet1-ky of cidr 10.240.10.0/16 and subnet subnet3-ky of cidr 10.240.30.0/24, both from VPC test-vpc1-ky, overlap in 10.240.30.0/24
+________________________________________________________________________________________________________________________________________________________________________________________________________
+
+"TCP Connections for which response is disabled" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+connection protocol: TCP dst-ports: 22 db-endpoint-gateway-ky[10.240.30.7] to vsi2-ky[10.240.20.4] both of VPC test-vpc1-ky is not responsive
```

Comment:
this sentence is not clear

Reply:
suggestions to improve:
- use notation vpc/vsi ... (it is not clear what "both of .." refers to)
- comma separation for various components in the sentence
- add "source:" , "destination: " 
- use term "TCP response is blocked" (the one that appears in explainability reports) instead of "not responsive"

Reply:
Agreed. I did some changes, hope it is clearer